server/services/interventionsService.ts

```diff
@@ -161,8 +161,8 @@ export interface ActionPlanAppointment {
 }
 
 export interface ActionPlanAppointmentUpdate {
-  appointmentTime: string
-  durationInMinutes: number
+  appointmentTime: string | null
```

Comment:
Hmm, didn't realise this was optional. Will need to update my PR after this is merged.

Reply:
I think it's always been optional in the model? All I've changed here is the update DTO.